src/test/java/org/opensearch/neuralsearch/plugin/NeuralSearchTests.java

```diff
@@ -117,6 +117,7 @@ public void testQuerySpecs() {
         assertTrue(querySpecs.stream().anyMatch(spec -> HybridQueryBuilder.NAME.equals(spec.getName().getPreferredName())));
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/neural-search/pull/1268")
```

Comment:
I am not entirely sure why this test is failing, will open a new issue

Reply:
https://github.com/opensearch-project/OpenSearch/pull/17611. This PR is the reason behind the test feature.

Reply:
ah you're right!

Reply:
There is an issue with onboarding to the new feature flag framework, see https://github.com/opensearch-project/OpenSearch/issues/17940. For the time being we are skipping this test